projects/graph/src/graph.py

```diff
@@ -9,11 +9,15 @@ def __init__(self):
         vertices = []
 
     def add_vertex(self, vertex):
-        vertex_plus_edge = {vertex, set()}
-        self.vertices.append(vertex_plus_edge)
+        self.vertices.append(Vertex(vertex))
 
     def add_edge(self, vertex_one, vertex_two):
         first_index = vertices.index(vertex_one)
         vertices[first_index] = {vertex_one, vertex_two}
         second_index = vertices.index(vertex_two)
         vertices[second_index] = {vertex_two, vertex_one}
+
+class Vertex:
+    def __init__(self, vertex, edge=set()):
+        self.vertex = vertex
+        self.edge = edge
```

Comment:
A small thing, but it might be nice to call this "edges" instead of simply "edge"